server.go

```diff
@@ -1713,67 +1609,21 @@ func (s *server) peerDoneHandler(sp *serverPeer) {
 	close(sp.quit)
 }
 
-// establishConn establishes a connection to the peer.
-func (s *server) establishConn(sp *serverPeer) error {
-	srvrLog.Debugf("Attempting to connect to %s", sp.Addr())
-	conn, err := btcdDial("tcp", sp.Addr())
-	if err != nil {
-		return err
-	}
-	sp.Connect(conn)
-	s.addrManager.Attempt(sp.NA())
-	return nil
-}
-
-// retryConn retries connection to the peer after the given duration.  It must
-// be run as a goroutine.
-func (s *server) retryConn(sp *serverPeer, initialAttempt bool) {
-	retryDuration := connectionRetryInterval
-	for {
-		if initialAttempt {
-			retryDuration = 0
-			initialAttempt = false
-		} else {
-			srvrLog.Debugf("Retrying connection to %s in %s", sp.Addr(),
-				retryDuration)
-		}
-		select {
-		case <-time.After(retryDuration):
-			err := s.establishConn(sp)
-			if err != nil {
-				retryDuration += connectionRetryInterval
-				if retryDuration > maxConnectionRetryInterval {
-					retryDuration = maxConnectionRetryInterval
-				}
-				continue
-			}
-			return
-
-		case <-sp.quit:
-			return
-
-		case <-s.quit:
-			return
-		}
-	}
-}
-
 // peerHandler is used to handle peer operations such as adding and removing
 // peers to and from the server, banning peers, and broadcasting messages to
 // peers.  It must be run in a goroutine.
 func (s *server) peerHandler() {
-	// Start the address manager and block manager, both of which are needed
-	// by peers.  This is done here since their lifecycle is closely tied
-	// to this handler and rather than adding more channels to sychronize
-	// things, it's easier and slightly faster to simply start and stop them
-	// in this handler.
+	// Start the address manager, block manager and connection manager, all of
```

Comment:
This comment update doesn't appear accurate any longer given the connection manager is not started until lower down.